src/cheri_insts.sail

```diff
@@ -596,6 +596,33 @@ function clause execute (CSetBoundsExact(cd, cs1, rs2)) = {
   RETIRE_SUCCESS
 }
 
+union clause ast = CSetBoundsRoundDown : (regidx, regidx, regidx)
+/*!
+ * Capability register *cd* is set to capability register *cs1* with its
+ * **base** field replaced with *cs1*.**address** and its **length** field
+ * replaced with the maximum exactly representable value no larger than
+ * the value in the integer register *rs2*.
```

Comment:
Add a note about the maximum length returned with the caveat that it may go away in future?